edx_event_bus_kafka/internal/consumer.py

```diff
@@ -274,6 +289,26 @@ def reset_offsets(consumer, partitions):
         finally:
             self.consumer.close()
 
+    def consume_indefinitely(self, offset_timestamp=None):
+        """
+        Consume events from a topic in an infinite loop.
+
+        Arguments:
+            offset_timestamp (datetime): Optional and deprecated; if supplied, call ``reset_offsets_indefinitely``
```

Comment:
Nit: The current wording made me think it was instructions for refactoring, and then repeated. But, now I get it.
```suggestion
            offset_timestamp (datetime): Optional and deprecated; if supplied, calls ``reset_offsets_indefinitely``
```

Reply:
Ah, good call.